### PR TITLE
Maxim kliashchou h2 t3

### PR DESCRIPTION
**Pull request by maximkliashchou**

### CODE REVIEW

**Review by Alexander-1313 — APPROVED** [review]
Lgfm

**Review by ArtikTico — APPROVED** [review]
gj

**Review by barilo-egor — APPROVED** [review]
lgfm

**Review by asraysun — APPROVED** [review]

File: students/Maxim_Kliashchou/HomeWork02/Task03/by.maxim.kliashchou/java/MathOperationsDemo.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    public static void main(String[] args) {
+        int number1 = 40, number2 = 12;
+
+        System.out.println("First number = " + number1);
+        System.out.println("Second number = " + number2);
+
+        System.out.println("Addition = " + (number1 + number2));
+        System.out.println("Substraction = " + (number1 - number2));
+        System.out.println("Multiplication = " + (number1 * number2));
+        System.out.println("Division = " + (number1 / number2));
```
Comment (javaGuruBY) [inline students/Maxim_Kliashchou/HomeWork02/Task03/by.maxim.kliashchou/java/MathOperationsDemo.java:11]: You'd better avoid performing operations in sout

**Review by javaGuruBY — APPROVED** [review]

**Pull request merged**